### PR TITLE
Add leaflet-canvas-markers in plugins.md

### PR DESCRIPTION
**Pull request by lipatoff**
Add leaflet-canvas-markers

### CODE REVIEW

**Comment by IvanSanchez** [comment]
The name is too similar to an already existing plugin, https://github.com/eJuke/Leaflet.Canvas-Markers :-/

**Comment by lipatoff** [comment]
> The name is too similar to an already existing plugin, https://github.com/eJuke/Leaflet.Canvas-Markers :-/

If I take name L.canvasMarker - Is it fine?

File: docs/plugins.md [hunk 1]
```diff
@@ -370,6 +370,15 @@ Plugins for loading basemaps or GIS raster layers in common (albeit non-default)
 		</td><td>
 			<a href="https://github.com/GeoportalPL">Geoportal Poland</a>
 		</td>
+    </tr>
+    <tr>
```
Comment (johnd0e) [inline docs/plugins.md:374]: Please use tabs

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Comment by Falke-Design** [comment]
We changed the design / structur for the plugin site. Now each plugin have a own file. Please reaname your plugin and then create a new PR #7805

**Pull request closed without merging**